Gopkg.toml

```diff
@@ -1,17 +1,15 @@
-required = ["github.com/Masterminds/semver"]
```

Comment:
for a while there (pre gps move) we didn't directly import it, and thus couldn't constrain it, so we kept getting the wrong version.